README.md

```diff
@@ -53,15 +53,12 @@ yarn start -- <minard-backend API URL> # then open http://localhost:3000/ in a b
 ```
 
 By default, when run locally, the minard-backend (charles) API runs
-on `http://localhost:8000`:
+on port 8000:
 
 ```shell
 yarn start -- http://localhost:8000
```

Comment:
Charles's endpoint should now be specified as `http://localtest.me:8000` as otherwise raw deployment authentication won't work